custom/ewsghana/reports/__init__.py

```diff
@@ -169,3 +174,25 @@ def is_reporting_type(self):
             if not location_type.administrative
         ]
         return sql_location.location_type in reporting_types
+
+
+class ProductSelectionPane(EWSData):
+    slug = 'product_selection_pane'
+    show_table = True
+    title = 'Product Selection Pane'
+
+    @property
+    def rows(self):
+        if self.config['program'] and not self.config['products']:
+            products = [product for product in SQLProduct.objects.filter(
+                program_id=self.config['program'], domain=self.config['domain'])]
+        elif self.config['program'] and self.config['products']:
+            products = [product for product in SQLProduct.objects.filter(
+                domain=self.config['domain'], product_id__in=self.config['products'])]
```

Comment:
`self.config['program']` is not referenced in this block, but it's required in the condition. Is that correct/intentional?